src/components/transactions/wallet/Swap/useSwapData.ts

```
@@ -0,0 +1,697 @@
+                            else if (fromAmount !== '') setFromAmount('');
+                        } else {
+                            if (details.outputAmount) setFromAmount(details.outputAmount.toSignificant(6));
+                            else if (fromAmount !== '') setFromAmount('');
+                        }
+                    }
+                }
+            } else {
+                // INSUFFICIENT LIQUIDITY OR NO TRADE FOUND
+                clearCalculatedDetails();
+                let errorMessage = 'Insufficient liquidity for this trade.';
+                if (details && details.route && !details.trade) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)